inc/Engine/Optimization/Minify/JS/AbstractJSOptimization.php

```diff
@@ -53,6 +53,7 @@ protected function get_excluded_files() {
 		$excluded_files[] = '/interactive-3d-flipbook-powered-physics-engine/assets/js/pdf.min.js';
 		$excluded_files[] = '/interactive-3d-flipbook-powered-physics-engine/assets/js/three.min.js';
 		$excluded_files[] = '/interactive-3d-flipbook-powered-physics-engine/assets/js/3d-flip-book.min.js';
+		$excluded_files[] = '/google-site-kit/dist/assets/js/googlesitekit-modules-(.*).js';
```

Comment:
@Tabrisrp Can we please extend this to all of Site Kit's JS files? We don't load any on the front end for non-logged-in users. The pattern here is too specific as well as it would not match our runtime. See the list of all the current assets here: https://plugins.svn.wordpress.org/google-site-kit/tags/1.49.0/dist/assets/js/

I haven't tested this, but this should work assuming regular expressions are supported 🙂 

```suggestion
		$excluded_files[] = '/google-site-kit/dist/.*\.js';
```

Reply:
I've moved on with excluding all JavaScript files.